README.md

```diff
@@ -2,31 +2,456 @@
 
 ## Learning Goals
 
-- Learning Goal 1
-- Learning Goal 2
+- Review one-to-many relationships and many-to-many relationships.
+- Learn how we can apply these relationships in Spring Boot.
 
 ## Introduction
 
-A 1-2 sentence summary of what will be covered.
+In the last module, we learned about the different relationships database tables
+can have with one another. Let's review the many-to-many relationship and the
+one-to-many relationships and then see how we can query these relationships in
+Spring Boot.
 
-## Topic 1
+## JPA Relationship Review
 
-Discuss the topic of the reading. Break into sections and sub-sections as
-appropriate to help with the reader's orientation with the material and future
-referencing.
+There are three kinds of relationships we talk about when describing database
+tables:
 
-## Topic 2
+- One-to-One: This is when one record in a table will relate to exactly one
+  other record in another table.
+  - Example: A country has exactly one capital city and each capital belongs
+    to exactly one country.
+- One-to-Many: This is when one record in a table can relate to multiple
+  records in another table.
+  - Example: An employee works for only one department but a department can
+    have many employees.
+- Many-to-Many: This is when multiple records in a table can relate to multiple
+  other records in another table.
+  - Example: A film may have many actors and an actor may be in many films.
 
-Discuss the topic of the reading. Break into sections and sub-sections as
-appropriate to help with the reader's orientation with the material and future
-referencing.
+The more common relationships we see are the one-to-many relationships and the
+many-to-many relationships. Let's see how to set up these relationships again
+in Spring Boot. Consider the following ER diagram:
 
-## Conclusion
+![football-ER-diagram](https://curriculum-content.s3.amazonaws.com/spring-mod-1/multiple-entities/football-team-and-football-player-schema.png)
+
+Let's modify our schema now in our sports database to add this new table and
+this many-to-many relationship:
```

Comment:
Should this say 'one-to-many'? (one team has many football players)

Reply:
Yes! Thanks for catching this.